portal/src/main/webapp/js/src/dynamicQuery.js

```diff
@@ -195,7 +195,7 @@ function loadMetaData() {
                 window.metaDataJson.cancer_studies[window.cancer_study_id_selected] = json;
                 //  Add Meta Data to current page
                 var username = $('#header_bar_table span').text() || '';
-                if(window.session_service_url !== '') {
+                if(vcSession.URL) {
```

Comment:
Actually got rid of session_service_url variable and to maintain consistency between files thought of using vcSession.URL
And in future if we are enabling virtual cohort we just need to delete sessionServiceUrl =""; line